Add note to include .exe when checking the Stan compiler on Windows

#### Summary:
The instructions for checking the compiler in the CmdStan User's Guide, section 1.5 Checking the Stan compiler, does not work in Windows as `.exe` need to be added.

The User's Guide states:
```
make examples/bernoulli/bernoulli    # on Windows use mingw32-make
```
However to compile the model on Windows one must write:
```
mingw32-make examples/bernoulli/bernoulli.exe
```
Note that `.exe` is needed.

#### Current Output:
```
mingw32-make examples/bernoulli/bernoulli
mingw32-make: *** No rule to make target 'examples/bernoulli/bernoulli'.  Stop.
```

#### Expected Output:
```
mingw32-make examples/bernoulli/bernoulli.exe
--- Translating Stan model to C++ code ---
bin/stanc.exe  --o=examples/bernoulli/bernoulli.hpp examples/bernoulli/bernoulli.stan

--- Compiling, linking C++ code ---
g++ -std=c++1y -m64 -D_REENTRANT -Wall -Wno-unused-function -Wno-uninitialized -Wno-unused-but-set-variable -Wno-unused-variable -Wno-sign-compare -Wno-unused-local-typedefs -Wno-int-in-bool-context -Wno-attributes -Wno-ignored-attributes      -I stan/lib/stan_math/lib/tbb_2019_U8/include   -O3 -I src -I stan/src -I lib/rapidjson_1.1.0/ -I stan/lib/stan_math/ -I stan/lib/stan_math/lib/eigen_3.3.7 -I stan/lib/stan_math/lib/boost_1.72.0 -I stan/lib/stan_math/lib/sundials_5.2.0/include  -D_USE_MATH_DEFINES  -DBOOST_DISABLE_ASSERTS        -c  -x c++ -o examples/bernoulli/bernoulli.o examples/bernoulli/bernoulli.hpp
g++ -std=c++1y -m64 -D_REENTRANT -Wall -Wno-unused-function -Wno-uninitialized -Wno-unused-but-set-variable -Wno-unused-variable -Wno-sign-compare -Wno-unused-local-typedefs -Wno-int-in-bool-context -Wno-attributes -Wno-ignored-attributes      -I stan/lib/stan_math/lib/tbb_2019_U8/include   -O3 -I src -I stan/src -I lib/rapidjson_1.1.0/ -I stan/lib/stan_math/ -I stan/lib/stan_math/lib/eigen_3.3.7 -I stan/lib/stan_math/lib/boost_1.72.0 -I stan/lib/stan_math/lib/sundials_5.2.0/include  -D_USE_MATH_DEFINES  -DBOOST_DISABLE_ASSERTS              -Wl,-L,"C:/Users/user/cmdstan-2.25.0/stan/lib/stan_math/lib/tbb" -Wl,-rpath,"C:/Users/user/cmdstan-2.25.0/stan/lib/stan_math/lib/tbb"      examples/bernoulli/bernoulli.o src/cmdstan/main.o  -static-libgcc -static-libstdc++       stan/lib/stan_math/lib/sundials_5.2.0/lib/libsundials_nvecserial.a stan/lib/stan_math/lib/sundials_5.2.0/lib/libsundials_cvodes.a stan/lib/stan_math/lib/sundials_5.2.0/lib/libsundials_idas.a stan/lib/stan_math/lib/sundials_5.2.0/lib/libsundials_kinsol.a  stan/lib/stan_math/lib/tbb/tbb.dll -o examples/bernoulli/bernoulli.exe
rm -f examples/bernoulli/bernoulli.o
```

#### Additional Information:
I'm running from cmd on Windows 10.

#### Current Version:
v2.25.0

--- a/src/cmdstan-guide/installation.Rmd
+++ b/src/cmdstan-guide/installation.Rmd
@@ -382,11 +382,11 @@ absolute path to the Intel TBB library when linking into Stan programs.)
 ## Checking the Stan compiler
 
 To check that the CmdStan installation is complete
-and in working order, run the following series of commands:
+and in working order, run the following series of commands on Linux/macOS:
 
 ```
 # compile the example
-> make examples/bernoulli/bernoulli    # on Windows use mingw32-make
+> make examples/bernoulli/bernoulli
 
 # fit to provided data (results of 10 trials, 2 out of 10 successes)
 > ./examples/bernoulli/bernoulli sample data file=examples/bernoulli/bernoulli.data.json
@@ -398,6 +398,20 @@ and in working order, run the following series of commands:
 # run the `bin/stansummary utility to summarize parameter estimates
 > bin/stansummary output.csv
 ```
+
+On Windows run the following:
+
+```
+# compile the example
+> mingw32-make examples/bernoulli/bernoulli.exe 
+
+# fit to provided data (results of 10 trials, 2 out of 10 successes)
+> ./examples/bernoulli/bernoulli.exe sample data file=examples/bernoulli/bernoulli.data.json
+
+# run the `bin/stansummary.exe utility to summarize parameter estimates
+> bin/stansummary.exe output.csv
+```
+
 The sample data in file `bernoulli.json.data` specifies 2 out of 10 successes, therefore
 the range `mean(theta)`$\pm$`sd(theta)` should include 0.2.
 

--- a/src/functions-reference/higher-order_functions.Rmd
+++ b/src/functions-reference/higher-order_functions.Rmd
@@ -44,7 +44,7 @@ this signature:
 
 ```
  vector algebra_system(vector y, vector theta,
-                              real[] x_r, int[] x_i)
+                              data real[] x_r, int[] x_i)
 ```
 
 The algebraic system function should return the value of the algebraic
@@ -67,32 +67,32 @@ through the algebraic system.
 
 ### Call to the algebraic solver
 
-<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i); -->
-\index{{\tt \bfseries algebra\_solver }!{\tt (function algebra\_system, vector y\_guess, vector theta, real[] x\_r, int[] x\_i): vector}|hyperpage}
+<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i); -->
+\index{{\tt \bfseries algebra\_solver }!{\tt (function algebra\_system, vector y\_guess, vector theta, data real[] x\_r, int[] x\_i): vector}|hyperpage}
 
-`vector` **`algebra_solver`**`(function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i)`<br>\newline
+`vector` **`algebra_solver`**`(function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i)`<br>\newline
 Solves the algebraic system, given an initial guess, using the Powell
 hybrid algorithm.
 
-<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i, real rel_tol, real f_tol, int max_steps); -->
-\index{{\tt \bfseries algebra\_solver }!{\tt (function algebra\_system, vector y\_guess, vector theta, real[] x\_r, int[] x\_i, real rel\_tol, real f\_tol, int max\_steps): vector}|hyperpage}
+<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i, data real rel_tol, data real f_tol, int max_steps); -->
+\index{{\tt \bfseries algebra\_solver }!{\tt (function algebra\_system, vector y\_guess, vector theta, data  real[] x\_r, int[] x\_i, data real rel\_tol, data real f\_tol, int max\_steps): vector}|hyperpage}
 
-`vector` **`algebra_solver`**`(function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i, real rel_tol, real f_tol, int max_steps)`<br>\newline
+`vector` **`algebra_solver`**`(function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i, data real rel_tol, data real f_tol, int max_steps)`<br>\newline
 Solves the algebraic system, given an initial guess, using the Powell
 hybrid algorithm with additional control parameters for the solver.
 
 *Note:* In future releases, the function `algebra_solver` will be deprecated
 and replaced with `algebra_solver_powell`.
 
-\index{{\tt \bfseries algebra\_solver\_newton }!{\tt (function algebra\_system, vector y\_guess, vector theta, real[] x\_r, int[] x\_i): vector}|hyperpage}
+\index{{\tt \bfseries algebra\_solver\_newton }!{\tt (function algebra\_system, vector y\_guess, vector theta, data real[] x\_r, int[] x\_i): vector}|hyperpage}
 
-`vector` **`algebra_solver_newton`**`(function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i)`<br>\newline
+`vector` **`algebra_solver_newton`**`(function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i)`<br>\newline
 Solves the algebraic system, given an initial guess, using Newton's method.
 
-<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i, real rel_tol, real f_tol, int max_steps); -->
-\index{{\tt \bfseries algebra\_solver\_newton }!{\tt (function algebra\_system, vector y\_guess, vector theta, real[] x\_r, int[] x\_i, real rel\_tol, real f\_tol, int max\_steps): vector}|hyperpage}
+<!-- vector; algebra_solver; (function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i, data real rel_tol, data real f_tol, int max_steps); -->
+\index{{\tt \bfseries algebra\_solver\_newton }!{\tt (function algebra\_system, vector y\_guess, vector theta, data real[] x\_r, int[] x\_i, data real rel\_tol, data real f\_tol, int max\_steps): vector}|hyperpage}
 
-`vector` **`algebra_solver_newton`**`(function algebra_system, vector y_guess, vector theta, real[] x_r, int[] x_i, real rel_tol, real f_tol, int max_steps)`<br>\newline
+`vector` **`algebra_solver_newton`**`(function algebra_system, vector y_guess, vector theta, data real[] x_r, int[] x_i, data real rel_tol, data real f_tol, int max_steps)`<br>\newline
 Solves the algebraic system, given an initial guess, using Newton's method
 with additional control parameters for the solver.
 

--- a/src/functions-reference/real-valued_basic_functions.Rmd
+++ b/src/functions-reference/real-valued_basic_functions.Rmd
@@ -1472,6 +1472,12 @@ of x and the natural exponentiation of y. \[
 `R` **`log_inv_logit`**`(T x)`<br>\newline
 natural logarithm of the inverse logit function of x
 
+<!-- R; log_inv_logit_diff; (T1 x, T2 y); -->
+\index{{\tt \bfseries log\_inv\_logit\_diff }!{\tt (T1 x, T2 y): R}|hyperpage}
+
+`R` **`log_inv_logit_diff`**`(T1 x, T2 y)`<br>\newline
+natural logarithm of the difference of the inverse logit function of x and the inverse logit function of y
+
 <!-- R; log1m_inv_logit; (T x); -->
 \index{{\tt \bfseries log1m\_inv\_logit }!{\tt (T x): R}|hyperpage}
 

--- a/src/stan-users-guide/algebraic-equations.Rmd
+++ b/src/stan-users-guide/algebraic-equations.Rmd
@@ -39,7 +39,7 @@ user-defined functions).
 ```
 vector system(vector y,        // unknowns
               vector theta,    // parameters
-              real[] x_r,      // data (real)
+              data real[] x_r,      // data (real)
               int[] x_i) {     // data (integer)
   vector[2] z;
   z[1] = y[1] - theta[1];

--- a/src/stan-users-guide/user-functions.Rmd
+++ b/src/stan-users-guide/user-functions.Rmd
@@ -638,10 +638,10 @@ A robust version of this function would test that the arguments are
 finite and that `sigma` is non-negative, e.g.,
 
 ```
-  if (is_nan(mu) || is_infinite(mu))
+  if (is_nan(mu) || is_inf(mu))
     reject("my_normal_rng: mu must be finite; ",
            "found mu = ", mu);
-  if (is_nan(sigma) || is_infinite(sigma) || sigma < 0)
+  if (is_nan(sigma) || is_inf(sigma) || sigma < 0)
     reject("my_normal_rng: sigma must be finite and non-negative; ",
            "found sigma = ", sigma);
 ```

